Gems/ScriptCanvas/Code/Include/ScriptCanvas/Core/Node.cpp

```diff
@@ -2391,7 +2391,22 @@ namespace ScriptCanvas
 
             if (variableIds.count(variableId) > 0)
             {
-                InitializeVariableReference(slot, variableIds);
+                //InitializeVariableReference(slot, variableIds);
+                slot.ClearVariableReference();
+                NodeNotificationsBus::Event(GetEntityId(), &NodeNotifications::OnSlotInputChanged, slot.GetId());
+
+                //ConvertSlotToValue(slot.GetId());
```

Comment:
Can the commented out code be deleted?

Reply:
oh that snuck in... wasn't supposed to be part of this PR, I'll revert it